libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -118,6 +118,34 @@ void AP_Mount_Backend::set_rate_target(float roll_degs, float pitch_degs, float
     }
 }
 
+void AP_Mount_Backend::control_mount_open_servo() {
+    // check if the open function is assigned to a servo channel.
+    uint8_t open_chan;
+    if (!SRV_Channels::find_channel(SRV_Channel::k_mount_open, open_chan) &&
+        !SRV_Channels::find_channel(SRV_Channel::k_mount2_open, open_chan)) {
```

Comment:
If you have two mounts, this will use the same servo channel for both.

Reply:
Not sure I'm getting what this is doing. I'd want both gimbals to move in sync sometimes, but not other times depending on what I'm doing. Is that what this is?

If that's what this is, it's a great idea, but is it optional?

Reply:
Yes, @nexton-winjeel is right.  We should check for k_mount_open or k_mount2_open depending upon the instance of the gimbal.